databus/src/main/java/com/bazaarvoice/emodb/databus/core/DatabusChannelConfiguration.java

```diff
@@ -13,7 +13,7 @@
 public class DatabusChannelConfiguration implements ChannelConfiguration {
     private static final Duration FANOUT_TTL = Duration.ofDays(365);  // Basically forever
     public static final Duration CANARY_TTL = Duration.ofDays(365);  // Basically forever
-    public static final Duration REPLAY_TTL = Duration.ofHours(50); // 2 days + 2 hours
+    public static final Duration REPLAY_TTL = Duration.ofHours(52); // 2 days + 4 hours
```

Comment:
why adding 2 more hrs here :?

Reply:
We are expanding the necessary buffer time of databus replay, so i added extra time for it to complete the extra work.